pycm/pycm_obj.py

```diff
@@ -166,11 +166,11 @@ def stat(
         """
         Print statistical measures table.
 
-        :param overall_param : overall parameters list for print, Example : ["Kappa","Scott PI]
+        :param overall_param: overall parameters list for print, Example: ["Kappa", "Scott PI"]
         :type overall_param : list
-        :param class_param : class parameters list for print, Example : ["TPR","TNR","AUC"]
+        :param class_param: class parameters list for print, Example: ["TPR", "TNR", "AUC"]
         :type class_param : list
-        :param class_name : class name (sub set of classes), Example :[1,2,3]
+        :param class_name : class name (a sub set of classes names), Example :[1, 2, 3]
```

Comment:
`:param class_name: class name (a sub set of classes names), Example :[1, 2, 3]`